poleno/attachments/utils.py

```diff
@@ -5,6 +5,18 @@
 from poleno import datacheck
 
 
+def attachment_file_is_working(attachment):
+    if not attachment.file:
```

Comment:
@martinmacko47 Nekontroloval som pripad ci `attachment.file.name` je prazdne. Takyto pripad by nemal nastat, resp. ak je to prazdny string potom `attachment.file` je `<FieldFile: None>` a padne to na tejto podmienke.

Reply:
To je OK. Malo by platit, ze `not attachment.file` je true prave vtedy ked `not attachment.file.name` je true